integration-tests/src/test/java/oracle/weblogic/kubernetes/ItTwoDomainsLoadBalancers.java

```diff
@@ -509,15 +592,71 @@ public void testApacheCustomSample() {
     logger.info("Verifying Apache custom sample");
     for (int i = 1; i <= numberOfDomains; i++) {
       int httpNodePort = getApacheNodePort(defaultNamespace, "http");
-      verifyClusterLoadbalancing(domainUids.get(i - 1), defaultNamespace, "http", httpNodePort, replicaCount,
+      verifyClusterLoadbalancing(domainUids.get(i - 1), "", "http", httpNodePort, replicaCount,
           false, "/weblogic" + i);
 
       int httpsNodePort = getApacheNodePort(defaultNamespace, "https");
-      verifyClusterLoadbalancing(domainUids.get(i - 1), defaultNamespace, "https", httpsNodePort, replicaCount,
+      verifyClusterLoadbalancing(domainUids.get(i - 1), "", "https", httpsNodePort, replicaCount,
           false, "/weblogic" + i);
     }
   }
 
+  /**
+   * Test verifies multiple WebLogic domains can be loadbalanced by NGINX loadbalancer with host based routing rules.
+   * Accesses the clusterview application deployed in the WebLogic cluster through NGINX host routing
+   * and verifies it is correctly routed to the specific domain cluster identified by the -H host header.
+   *
+   */
+  @Order(13)
+  @Test
+  @DisplayName("verify NGINX host routing with HTTP across two domains")
+  public void testNginxHttpHostRoutingAcrossDomains() {
+
+    // verify load balancing works when 2 domains are running in the same namespace
+    logger.info("Verifying NGINX load balancer host routing");
+    for (String domainUid : domainUids) {
+      verifyClusterLoadbalancing(domainUid, domainUid + "." + defaultNamespace + ".nginx.nonssl.test",
+          "http", getNginxLbNodePort("http"), replicaCount, true, "");
+    }
+  }
+
+  /**
+   * Test verifies multiple WebLogic domains can be loadbalanced by NGINX loadbalancer with host based routing rules.
+   * Accesses the clusterview application deployed in the WebLogic cluster through NGINX tls host routing
+   * and verifies it is correctly routed to the specific domain cluster identified by the -H host header.
+   *
+   */
+  @Order(14)
+  @Test
+  @DisplayName("verify NGINX tls host routing across two domains")
+  public void testNginxHttpsHostRoutingAcrossDomains() {
+
+    // verify load balancing works when 2 domains are running in the same namespace
+    logger.info("Verifying NGINX tls host routing");
+    for (String domainUid : domainUids) {
+      verifyClusterLoadbalancing(domainUid, domainUid + "." + defaultNamespace + ".nginx.ssl.test",
+          "https", getNginxLbNodePort("https"), replicaCount, true, "");
+    }
+  }
+
+  /**
+   * Test verifies multiple WebLogic domains can be loadbalanced by NGINX loadbalancer with path routing rules.
+   * Accesses the clusterview application deployed in the WebLogic cluster through NGINX loadbalancer and verifies it
+   * is correctly routed to the specific domain cluster.
+   */
+  @Order(15)
+  @Test
+  @DisplayName("Load balance WebLogic cluster traffic through NGINX load balancer path routing")
```

Comment:
Provide consistent description for both host/path routing test methods as follows ...

  @DisplayName("verify NGINX path routing with HTTP protocol across two domains")

Reply:
fixed